src/monexamine.cpp

```diff
@@ -0,0 +1,25 @@
+#include "monexamine.h"
+#include "monster.h"
+#include "string_input_popup.h"
+#include "monster.h"
+#include "calendar.h"
+#include "game.h"
+#include "map.h"
+#include "string_input_popup.h"
+#include "messages.h"
+
+const efftype_id effect_milked( "milked" );
+
+void monexamine::milk_source( monster *source_mon )
+{
+    item milk( "milk", 0, 1 );
+    if( !source_mon->has_effect( effect_milked ) ||
```

Comment:
You shouldn't need this case: if the creature has no effect of a particular ID, it has 0 duration of said effect, meaning it's handled by case below.